src/main/java/Howie.java

```diff
@@ -0,0 +1,40 @@
+import commands.Command;
+import exceptions.DukeException;
+import parser.Parser;
+import storage.Storage;
+import task.Task;
+import tasklists.TaskList;
+import ui.Ui;
+
+import java.util.List;
+
+/**
+ * The chat-bot To-Do program named Howie which keeps tracks of major major tasks such as
+ * todo, event and deadlines.
+ * @author Koo Zhuo Hui
+ */
+public class Howie {
+
+    /**
+     * Initialises the program.
+     * @param args Input arguments.
+     * @throws Exception Throws DukeException and IOException when invalid commands are entered
+     * or input exception occurs.
+     */
+    public static void main(String[] args) throws Exception {
+        Ui ui = new Ui();
+        Storage storage = new Storage();
+        TaskList taskLs = storage.readFile();
```

Comment:
`taskLs` may not be an ideal name. Maybe `taskList`.  
I'm not sure if `Ls` is a short form of `List`?

https://se-education.org/guides/conventions/java/intermediate.html#naming

Reply:
Agree with Camille, taskList might be better naming :)

Reply:
Thanks for pointing it out! Would proceed to rectify 👍🏻